renderpyg/sprite.py

```diff
@@ -64,18 +67,18 @@ def __init__(self, source, width=None, height=None, **kwargs):
 			grid
 		"""
 		pg.sprite.Sprite.__init__(self)
-		if type(source) == Texture:
+		if isinstance(source, Texture):
 			self.images = fetch_images(source, width, height, **kwargs)
-		elif hasattr(source, '__len__') and type(source[0]) == Renderer:
+		elif hasattr(source, '__len__') and isinstance(source[0], Renderer):
 			self.images = load_images(
 				source[0], source[1], width, height, **kwargs)
-		elif hasattr(source, '__len__') and type(source[0]) == Image:
+		elif hasattr(source, '__len__') and isinstance(source[0], Image):
 			self.images = []
 			for image in source:
 				self.images.append(Image(image))
 		else:
 			raise ValueError(
-				'Cannot parse {} as source of GPUAniSPrite'.format(source))
+				f'Cannot parse {source} as source of GPUAniSPrite')
```

Comment:
rather not require python 3.6 but please feel free to try convincing me.

Mainly, I don't want to build python from source in the stream runtime. I thought fstrings came in 3.7 and not sure how old 3.6 is. I know my Mint 18.3 used python 3.5. Not sure how common that is.

Reply:
Python 3.5 has reached EOL and will no longer receive anything but bug fixes. It’s almost always going to be better to just require 3.6 unless one of your users needs 3.5 for some reason.